.github/workflows/remote-build-android.yml

```
@@ -59,3 +59,11 @@ jobs:
           variant: ${{ matrix.variant }}
           rock-build-extra-params: '--extra-params -PreactNativeArchitectures=arm64-v8a,x86_64'
           comment-bot: false
+
+      - name: Verify ELF alignment
+        run: |
+          APK_PATH=$(find android/app/build/outputs/apk -type f -name "*.apk" | head -n 1)
+          echo "Using APK: $APK_PATH"
+
+          chmod +x scripts/check-elf-alignment.sh
+          scripts/check-elf-alignment.sh "$APK_PATH"
```

Missing trailing newline:


```suggestion
          scripts/check-elf-alignment.sh "$APK_PATH"

```